.github/scripts/mixed_line_ending.py

```diff
@@ -0,0 +1,38 @@
+"""mixed_line_ending.py"""
```

Comment:
This one doesn't seem necessary since we already have git configured to use LF.

Reply:
Good argument